pandapower/pf/create_jacobian_facts.py

```diff
@@ -184,3 +183,376 @@ def create_J_modification_tcsc(V, Ybus_tcsc, x_control, svc_controllable, tcsc_c
     J_m = csr_matrix(J_m)
 
     return J_m
+
+#
+# def create_J_modification_ssc(V, Ybus_ssc, x_control,
```

Comment:
Should we keep this old part as a comment?

Reply:
removed it